src/vm/arm64/CallDescrWorkerARM64.asm

```diff
@@ -76,6 +77,8 @@ LNoFloatingPoint
         ldr     x9, [x19,#CallDescrData__pTarget]
         blr     x9
 
+        ldr     x19, [fp, 24] ; Fixup corrupted X19 callee preserved register
```

Comment:
Yes this is a workaround for unexpected x19 corruption.  I had tried to debug this failure by storing the correct value for x19 on the stack above and adding a conditional break, but was unable to do so.  The test seemed to fail to make forward progress.

Let me try to reproduce.  I may have better luck as the GC is more stable now.